### PR TITLE
feat(Фугапедия): new version

### PR DESCRIPTION
**Pull request by Syjalo**
## Description 
The presence was modified due to changes in the site design.

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>

image
image
image

<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/#/Фугапедия/presence.ts [hunk 1]
```diff
@@ -4,250 +4,172 @@ const presence = new Presence({
...
+		}
 	}
 
 	presence.setActivity(presenceData);
 });
+
+function parseName(start: number): string;
+function parseName(id: string): string;
+function parseName(start: number, id: string): string;
+function parseName(start: number | string, id?: string): string {
```
Comment (ImRodry) [inline websites/#/Фугапедия/presence.ts:162]: all these overloads return the same thing, making them useless. You can just remove them and keep the general declaration

File: websites/#/Фугапедия/presence.ts [hunk 2]
```diff
@@ -4,250 +4,172 @@ const presence = new Presence({
...
+				if (splittedPathname[2] === "me") {
+					presenceData.details = "Смотрит свой профиль";
+					presenceData.state = document.querySelector(
+						"body > div > div > main > div > div > div > div > div > h2.font-serif-text"
+					).textContent;
+				} else presenceData.state = parseName("profile_user_id");
+				presenceData.largeImageKey = document.querySelector<HTMLImageElement>(
+					"body > div > div > main > div > div > div > div > div > img"
+				).src;
+				presenceData.smallImageKey = "logo";
```
Comment (ImRodry) [inline websites/#/Фугапедия/presence.ts:135]: these should be swapped around

**Review by ImRodry — CHANGES_REQUESTED** [review]

File: websites/#/Фугапедия/presence.ts [hunk 3]
```diff
@@ -4,250 +4,172 @@ const presence = new Presence({
...
+		const { value } = generalSearch;
+		if (!privacyMode && value) presenceData.state = `по запросу «${value}»`;
 		presence.setActivity(presenceData);
 		return;
 	}
 
-	switch (document.location.pathname) {
-		case "/": {
+	switch (splittedPathname[1]) {
+		case "": {
```
[the remark this answers is not in the document]
Comment (Syjalo) [reply websites/#/Фугапедия/presence.ts:28]: I split it by "/". The strings can't contain this in any way. It works, the proof is in the screenshots.

File: websites/#/Фугапедия/presence.ts (hunk 2 above)
Comment (Syjalo) [reply websites/#/Фугапедия/presence.ts:135]: pfp can't be in the large image?

File: websites/#/Фугапедия/presence.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (ImRodry) [reply websites/#/Фугапедия/presence.ts:28]: Can you show me the value of your array logged when this case is true?

File: websites/#/Фугапедия/presence.ts (hunk 2 above)
Comment (ImRodry) [reply websites/#/Фугапедия/presence.ts:135]: Large image should always be the service logo

Comment (ImRodry) [reply websites/#/Фугапедия/presence.ts:135]: Large image should always be the service logo

File: websites/#/Фугапедия/presence.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (Syjalo) [reply websites/#/Фугапедия/presence.ts:28]: image

File: websites/#/Фугапедия/presence.ts (hunk 2 above)
Comment (Syjalo) [reply websites/#/Фугапедия/presence.ts:135]: But some presences use it not only for the logo. And I didn't find any info that it should always be the service logo.

**Review by EncryptedDev — APPROVED** [review]
LGTM

**Review by ImRodry — APPROVED** [review]

**Pull request merged**